app/src/main/kotlin/com/wire/android/migration/feature/MigrateClientsDataUseCase.kt

```diff
@@ -43,13 +44,20 @@ class MigrateClientsDataUseCase @Inject constructor(
 
             // add registered client id, sync will start when the registered id is persisted
             coreLogic.sessionScope(userId) {
-                when (val result = this.client.persistRegisteredClientIdUseCase(clientId)) {
-                    is PersistRegisteredClientIdResult.Failure.Generic ->
+                when (val result = this.client.importClient(clientId, RegisterClientParam(
+                    password = null,
+                    capabilities = null
+                ))) {
```

Comment:
I think short comment with explanation would be beneficial here as not everyone will immediately understand why nulls are passed here 😄